block/details.go

```diff
@@ -0,0 +1,10 @@
+package block
```

Comment:
Not sure why this package is named "block". It seems like it would be more directly tied to a workspace rather than a block. Or maybe this use of "block" is different than a block from furion.